integration/ig/k8s/trace_dns_test.go

```diff
@@ -225,6 +222,10 @@ func TestTraceDnsHost(t *testing.T) {
 
 			normalize := func(e *dnsTypes.Event) {
 				e.Timestamp = 0
+				// nslookup has several threads, so we prefer to normalize the command
+				// as it can be hard to guess which thread (nslookup, isc-net-0000,
+				// isc-timer or isc-socket-0) will do the DNS request.
+				e.Comm = ""
```

Comment:
I suppose that by design it's always isc-worker0000 doing the request? I have tried multiple times and I always get the same thread.